extensions/wikia/Wall/WallExternalController.class.php

```diff
@@ -268,11 +290,11 @@ public function deleteMessage() {
 		switch( $this->request->getVal( 'mode' ) ) {
 			case 'rev':
 				if ( $mw->canDelete( $this->wg->User ) ) {
-					$result = $mw->delete( wfMsgForContent( 'wall-delete-reason' ), true );
+					$result = $mw->delete( wfMessage( 'wall-delete-reason' )->inContentLanguage()->escaped(), true );
 					$this->response->setVal( 'status', $result );
 					return true;
 				} else {
-					$this->response->setVal( 'error', wfMsg( 'wall-message-no-permission' ) );
+					$this->response->setVal( 'error', wfMessage( 'wall-message-no-permission' )->escaped() );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wall-message-no-permission" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)